index.html

```diff
@@ -16,7 +16,11 @@ <h1 class="ui inverted header">
     </h1>
     <div class="ui search">
 		  <div class="ui fluid left icon input">
-		    <input class="prompt" placeholder="{{ page.searchExamples | join: ', '}}…" type="search">
+		    <input class="prompt"
+              placeholder="{{ page.searchExamples | join: ', '}}…"
+              onfocus="this.placeholder = ''"
+              onblur="this.placeholder = '{{ page.searchExamples | join: ', '}}…'"
```

Comment:
This is [really bad](http://stackoverflow.com/questions/8392374/difference-between-obtrusive-and-unobtrusive-javascript). Please [use CSS](http://stackoverflow.com/questions/12821746/clear-html5-placeholder-attribute-text-on-focus-in-chrome).

> And really, this is a workaround a flaw in Semantic UI. Placeholder text shouldn't become darker upon selection like it does, and it shouldn't be up to you to redefine browser behaviour.